src/modules/keyboardmanager/test/KeyboardManagerRemapLogicTest.cpp

```diff
@@ -144,6 +144,72 @@ namespace KeyboardManagerRemapLogicTests
             Assert::AreEqual(mockedInputHandler.GetVirtualKeyState(0x41), false);
             Assert::AreEqual(mockedInputHandler.GetVirtualKeyState(VK_LWIN), false);
         }
+
+        // Test if SendVirtualInput is sent exactly once with the suppress flag when Caps Lock is remapped to Ctrl
+        TEST_METHOD (HandleSingleKeyRemapEvent_ShouldSendVirutalInputWithSuppressFlagExactlyOnce_WhenCapsLockIsMappedToCtrlAltShift)
+        {
+            // Reset test environment
```

Comment:
nit:  This looks like a candidate for `TEST_METHOD_INITIALIZE` or cleanup.

Reply:
Great suggestion, I didn't know about `TEST_METHOD_INITIALIZE`. I'll make that change for all the tests to reduce the amount of duplicated code.